### DISCUSSION TITLE
About result after changes into a ROM folder.

### DISCUSSION THREAD

**Discussion by craft-and-code** [emmercm/igir#851] [Ideas]
Hello,

I just use this command: `npx igir move clean test`

And I got this result:

```
✓ Scanning for DATs ·········· | 1 unique DAT found
✓ Scanning for ROMs ·········· | 89 files found
✓ Brand - Model ··············· | 84/85 games, 1/1 BIOSes, 78/78 retail releases written
✓ Cleaning output directory ·· | 5 files recycled
```

About 5 files recycled, I just thinking it's shame we do not have more information when something is changing (changing name for exemple) or moving into the trash. I don't think so I saw an option to get igir more expressive.

Would be nice to get something more tasty, what do you think?

EDIT: I was 85/85 before the update of this DAT file, so something was deleted, I have no idea what it is...

**Comment by emmercm** [comment]
RE: more information about ROM writing, you can increase the log verbosity level (i.e. `-vvv`) which will give you a lot of output, probably more than you want. I haven't come up with a better way to show any of this information.

**Comment by emmercm** [comment]
RE: more information about input cleaning, the `report` command will write this information to its CSV. Trace logs (`-vvv`) don't currently output this information, but they could if people would find it valuable.
